src/views/template/factory/factory.html

```diff
@@ -32,16 +32,9 @@
                 {{#l}}Place order{{/l}}
             </button>
         </form>
-        <form name="frequencyTrucks" action="">
-            <button type="submit" class="btn btn-success pull-right margin-top-10">
-                {{#1}} Buy an extra truck{{/1}}
-            </button>
-        </form>
-        <form name="frequencyTrucks" action="">
-            <button type="submit" class="btn btn-success pull-right margin-top-10">
-                {{#1}} Buy an extra truck{{/1}}
-                {{#1}} Buy an extra truck{{/1}}
-            </button>
-        </form>
+        <form name="frequencyTrucks" action"">
```

Comment:
This line needs to go. There is no need for a form here - it is just a button group.